### PR TITLE
Convert to PHP arrays for routing config

### LINKED ISSUE #637: Move from JSON to PHP array for route declaration
Yes, it's a minuscule performance improvement for what we're doing, but PHP arrays can be cached in opcache (and since the routes table is static it'll be cached once across all requests), while JSON blobs have to be read from disk every time. Would be easy enough to swap things over and we don't use the routes file in its JSON form anywhere else from what I can see.

### PR DESCRIPTION
**Pull request by dstockto**
Fixes route for creating comments on events
Closes #637

### CODE REVIEW

File: src/config/routes/2.1.php [hunk 1]
```diff
@@ -0,0 +1,678 @@
...
+        'controller' => EventsController::class,
+        'action'     => 'rejectAction',
+        'verbs'      =>
+            [
+                'DELETE',
+            ],
+    ],
+    [
+        'path'       => '/events(/[^/]+)*/comments',
+        'controller' => EventCommentsController::class,
```
Comment (dstockto) [inline src/config/routes/2.1.php]: This is corrected from the json route file.

File: src/config/routes/2.1.json [hunk 2]
```diff
@@ -1,458 +0,0 @@
...
-    },
-    {
-        "path": "/events/(\\d+)/approval$",
-        "controller": "Joindin\\Api\\Controller\\EventsController",
-        "action": "rejectAction",
-        "verbs": ["DELETE"]
-    },
-    {
-        "path": "/events(/[^/]+)*/comments",
-        "controller": "Joindin\\Api\\Controller\\Event_commentsController",
```
Comment (dstockto) [inline src/config/routes/2.1.json]: This is the broken part of the old config.

**Comment by ericpoe** [comment]
@dstockto have you had a chance to look into that merge conflict?

**Comment by dstockto** [comment]
@ericpoe I'll take a look - it was not conflicting when it was PR'd 3 weeks ago. I don't think github notifies when it starts conflicting.

**Comment by zghosts** [comment]
@dstockto You might want to incorporate #684 into this PR

**Comment by dstockto** [comment]
Agreed, @zghosts  - I think that's the cause of the conflict.

**Comment by dstockto** [comment]
@mcneely @ericpoe @zghosts - This is no longer conflicting and includes the changes to routing from #684

**Pull request merged**